src/sack.c

```diff
@@ -835,7 +835,6 @@ hy_sack_add_excludes(HySack sack, HyPackageSet pset)
 	map_init(excl, pool->nsolvables);
 	sack->pkg_excludes = excl;
     }
-    assert(excl->size >= nexcl->size);
```

Comment:
IMO it should be `assert(excl->size =< nexcl->size);` Although you should not get it fired if you use the hawkey sack workflow correctly. e.g.: load all repos to sack, applyexcludes and then use of queries, selectors and goals.